src/main/java/com/gmail/yuramitryahin/model/Role.java

```diff
@@ -0,0 +1,28 @@
+package com.gmail.yuramitryahin.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
What are these?

Reply:
Lombok annotation for automatic building of getters, setters, toString and constructs